src/server/CodeStyle/FormattingVisitor.py

```
@@ -119,8 +132,22 @@ def visitMethodDeclaration(self, ctx: JavaParser.MethodDeclarationContext):
         method_signature = f"{' '.join(modifiers)} {return_type} {method_name}".strip()
 
         if grandparent: 
-            self.rewriter.replaceRangeTokens(grandparent.start, ctx.identifier().stop, f"\n{self._get_indent()}{method_signature}")
+            # Check if this is the first method in the class
+            is_first_method = True
+            for child in grandparent.parentCtx.children:
+                if isinstance(child, JavaParser.ClassBodyDeclarationContext):
+                    if child.memberDeclaration() and isinstance(child.memberDeclaration(), JavaParser.MethodDeclarationContext):
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)